common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -56,7 +56,7 @@ var _ p.ExecutionStore = (*cassandraPersistence)(nil)
 // Where x is any hexadecimal value, E represents the entity type valid values are:
 // E = {DomainID = 1, WorkflowID = 2, RunID = 3}
 // R represents row type in executions table, valid values are:
-// R = {Shard = 1, Execution = 2, Transfer = 3, Timer = 4, Replication = 5}
+// R = {Shard = 1, Execution = 2, Transfer = 3, Timer = 4, Replication = 5, DLQ = 6, CrossCluster = 7}
```

Comment:
Thanks for adding this. Could we update the DLQ to Replication_DLQ

Reply:
Done.